indico/modules/designer/templates/_list.html

```diff
@@ -60,9 +68,43 @@
                 {% endif %}
             {% endif %}
         </td>
+        <td>
+            {% if tpl.registration_form %}
+                {{- pgettext('Designer template linked to registration form', 'Linked to') -}}
+                <a href="{{ url_for('event_registration.manage_regform', tpl.registration_form) }}">
+                    {{ tpl.registration_form.title }}
+                </a>
+                <button class="i-button icon-cross icon-only text-color borderless danger"
+                        title="{% trans %}Unlink registration form{% endtrans %}"
+                        data-href="{{ url_for('.unlink_regform', tpl) }}"
+                        data-method="PATCH"
+                        data-update="#template-list"
+                        data-confirm="{% trans title=tpl.registration_form.title -%}
+                                            Are you sure you want to unlink the registration form '{{ title }}'?
+                                            Any placeholders specific to that registration form will be removed
+                                            from the template.
+                                      {%- endtrans %}">
+                </button>
+            {% endif %}
+        </td>
         <td>
             <div class="thin toolbar right">
                 <div class="group">
+                    {% if not tpl.registration_form and linkable_forms[tpl] %}
+                        <button class="i-button icon-plus icon-only highlight text-color js-dropdown borderless"
```

Comment:
What about using an icon more indicative to what this does... Was thinking either `icon-link` or `icon-clipboard` (which we use for regforms). What do you think?
![image](https://github.com/indico/indico/assets/27357203/c944c4b9-5c6d-46cc-8505-67634d2f9fbb)

Reply:
I went with the feather icon (`icon-quill`) in the end since that's what we use in most places for regforms